tests/simulator/refinement/harris.py

```
@@ -0,0 +1,225 @@
+    return datahier
+
+
+def get_hier(path):
+    return get_time(path)
+
+
+def post_advance(new_time):
+    datahier = check_hier(
+        get_hier("/home/aunai/Documents/code/phare/phare_jobs/run104/diags_master")
+    )
+    errors = test.base_test_overlaped_fields_are_equal(datahier, new_time)
```

Ruff

Local variable `errors` is assigned to but never used

Remove assignment to unused variable `errors`

(F841)